web/index.html

```diff
@@ -58,7 +58,8 @@
 }
 
 #invisible_click {
-	position: fixed;	
+	position: fixed;
+	opacity: 0;
```

Comment:
This seems to fix the weird dark blue border that would sometimes show up on mobile.